components/button/floating-buttons.md

```diff
@@ -19,7 +19,7 @@ Floating workflow buttons behavior can be added by using the `<d2l-floating-butt
 <p>
   Lorem ipsum dolor sit amet, consectetur adipisicing elit. Adipisci repellat cum totam! Enim, sunt. Numquam voluptate, velit quisquam ipsa molestias laudantium odit reiciendis nisi corporis voluptatibus, voluptatum sunt natus, accusantium magnam consequatur fugit officiis minima voluptatem consequuntur nam, earum necessitatibus! Cupiditate ullam repellendus, eius iure voluptas at commodi consectetur, quia, adipisci possimus, ex mollitia. Labore harum error consectetur officiis aut optio, temporibus iste nobis ducimus cumque laudantium rem pariatur. Ut repudiandae id, consequuntur quasi quis pariatur autem corporis perferendis facilis eius similique voluptatibus iusto deleniti odio officia numquam tenetur excepturi, aspernatur sunt minima aut fugiat ipsam.
 </p>
-<d2l-floating-buttons min-height="100px">
+<d2l-floating-buttons>
```

Comment:
@GeorgePlukov I think this might break the demo in the Daylight site... do you think adding `always-float` would fix things?

Reply:
I should be able to get the demos working with the `always-float` property. Since `min-height` has a default value set is it not possible this changes the point where buttons start floating on some pages?

Reply:
Ok thanks -- I've updated it to set `always-float`.

It shouldn't cause any change in behaviour since nobody was setting the attribute anywhere, so it was always `500px`.